src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIRangeGuard.java

```diff
@@ -19,6 +21,9 @@
 import net.minecraft.world.DifficultyInstance;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)